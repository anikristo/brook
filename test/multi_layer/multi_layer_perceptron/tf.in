12 2
-27 -3
1 2
4 5
6 7
8 9
10 11
13 4
-2 0
5 1
2 7
23 9
3 11