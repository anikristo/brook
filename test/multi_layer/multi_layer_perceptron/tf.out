12 2
21.1779 2118.65
3.54775 59.7973
12.3181 399.846
19.3907 570.032
29.2397 884.727
48.605 1096.76
53.1605 480.921
10.4218 1367.37
17.5087 441.884
60.2501 555.278
42.1974 1483.06
28.1237 1735.4
